src/status_im/extensions/capacities/components.cljs

```diff
@@ -25,26 +28,44 @@
     (js/clearTimeout id))
   (reset! current (js/setTimeout #(on-input-change-text on-change value) delay)))
 
-(defn input [{:keys [keyboard-type style error on-change change-delay placeholder placeholder-text-color selection-color
+(defn- in-range
+  ;; Determines whether a number is in a range
+  [n, start, end]
```

Comment:
Commas are a bit weird here :)

Reply:
No more commas at all 👍

Reply:
I still see them?

Reply:
@jeluard it was removed in the last commit: https://github.com/status-im/status-react/pull/8229/commits/0ce6e99147d16cfd12f0e5229a7f03eb15d26cca

Reply:
they are still here

Reply:
commas still here